src/test/regress/expected/multi_router_planner_fast_path.out

```diff
@@ -1994,12 +1994,13 @@ DEBUG:  query has a single distribution column value: 1
  41 |         1 | aznavour     |      11814
 (5 rows)
 
+-- Temporarily reduce the logging level because PG17 emits additional info
+-- for CREATE MATERIALIED VIEW xx AS
+-- https://git.postgresql.org/gitweb/?p=postgresql.git;a=commitdiff;h=b4da732fd64
+set client_min_messages to DEBUG1;
 -- materialized views can be created for fast-path router plannable queries
 CREATE MATERIALIZED VIEW mv_articles_hash_empty AS
 	SELECT * FROM articles_hash WHERE author_id = 1;
-DEBUG:  Distributed planning for a fast-path router query
-DEBUG:  Creating router plan
-DEBUG:  query has a single distribution column value: 1
```

Comment:
This debug is actually useful to see.

Also, we see the extra pg_temp debugs in 3 tests:
`multi_router_planner`, `multi_mx_router_planner`, `multi_router_planner_fast_path`
I think we qualify for a normalize rule here :) Something like:
`/DEBUG:  drop auto-cascades to type(.*)pg_temp_(.*)/d`

Reply:
Fair enough.. this would delete lines matching the PG17 DEBUG message from all .out files, right?

Thanks for the suggestion! Agreed that its good to preserve the existing DEBUG messages.

Reply:
> this would delete lines matching the PG17 DEBUG message from all .out files

Hopefully yes